galsim/compound.py

```diff
@@ -58,7 +58,7 @@ def Add(*args, **kwargs):
     @returns a Sum or ChromaticSum instance as appropriate.
     """
     if len(args) == 0:
-        raise ValueError("At least one ChromaticObject or GSObject must be provided.")
+        raise TypeError("At least one ChromaticObject or GSObject must be provided.")
```

Comment:
I think this corresponds to what Python normally does for the wrong number of arguments.  I don't particularly like this convention, since it doesn't seem to be related to types, but there isn't really a better choice AFAICT.

IMO, there should also be a standard exception called ArgumentError or ParameterError.  Something like that.  But these don't exist, and every time one of these is what I think I should use, python convention seems to be to use TypeError instead.